newrelic/hooks/mlmodel_openai.py

```diff
@@ -676,6 +678,8 @@ async def wrap_chat_completion_async(wrapped, instance, args, kwargs):
     # Framework metric also used for entity tagging in the UI
     transaction.add_ml_model_info("OpenAI", OPENAI_VERSION)
     transaction._add_agent_attribute("llm", True)
+    if not settings.ai_monitoring.streaming.enabled:
+        transaction.record_custom_metric("Supportability/Python/ML/Streaming/Disabled", 1)
```

Comment:
The way I had been thinking this supportability metric would be reported was by sending it once when we connect to the data collector in `core/application.py`. This is how we report the enabled/ disabled metrics for logging as well: https://github.com/newrelic/newrelic-python-agent/blob/main/newrelic/core/application.py#L555.